### ISSUE TITLE
Firefox: "TypeError: CSS2Properties doesn't have an indexed property setter for '0'"

### ISSUE THREAD

**Issue by sneurlax** [fabiandev/css-animator#21]
Tested in Firefox 52.3.0 and 56.0 (64-bit)

When an animation ends I'm getting:

```
ERROR TypeError: CSS2Properties doesn't have an indexed property setter for '0'
Stack trace:
["../../../../css-animator/builder/animation_builder.js"]/AnimationBuilder.prototype.removeStyles@http://localhost:4200/vendor.bundle.js:86198:17
["../../../../css-animator/builder/animation_builder.js"]/AnimationBuilder.prototype.reset@http://localhost:4200/vendor.bundle.js:85964:9
["../../../../css-animator/builder/animation_builder.js"]/AnimationBuilder.prototype.animate/</delay<@http://localhost:4200/vendor.bundle.js:85942:17
["../../../../zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask@http://localhost:4200/polyfills.bundle.js:3004:17
forkInnerZoneWithAngularBehavior/zone._inner<.onInvokeTask@http://localhost:4200/vendor.bundle.js:261263:24
["../../../../zone.js/dist/zone.js"]/</</ZoneDelegate.prototype.invokeTask@http://localhost:4200/polyfills.bundle.js:3003:17
["../../../../zone.js/dist/zone.js"]/</</Zone.prototype.runTask@http://localhost:4200/polyfills.bundle.js:2771:28
ZoneTask/this.invoke@http://localhost:4200/polyfills.bundle.js:3066:28
timer@http://localhost:4200/polyfills.bundle.js:4270:17
  vendor.bundle.js:258402:5
```

and promises end prematurely, neither resolving nor rejecting.

**Comment by fabiandev** [comment]
Thanks for flagging this, I'll have a look asap.

**Comment by orla11** [comment]
I confirm the error on Firefox

**Comment by fabiandev** [comment]
I'm on it guys. Weird behavior of Firefox returning `CSS2Properties` rather than `CSSStyleDeclaration` from `HTMLElement.style`.

**Comment by fabiandev** [comment]
@orla11 @sneurlax Can you confirm that this is fixed as of `v2.2.0-beta.3`? You can install the beta via `css-animator@next`.

**Comment by fabiandev** [comment]
This is now published as of `v2.2.0`.

**Comment by fabiandev** [comment]
Release `2.2.0` wasn't published correctly to `npm`, however, this should be fixed as of `v2.2.1`.
